### DISCUSSION TITLE
Make the `{% script %}` tag work inside `{% cache %}` tags

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#11243] [Ideas] [answered]
It'd be great if the `{% cache %}` tag would register any external or inline scripts added using the `{% script %}` tag, similar to how it handles the `{% js %}` and `{% css %}` tags.

**Comment by brandonkelly** [comment, marked as answer]
It does, as of 3.7.0 :)

**Reply by mmikkel** [reply]
Huh! I swear I tested it before, but I must've messed up somewhere 🙃 Anyway, thanks!
